gen-env-types.js

```diff
@@ -75,14 +78,18 @@ const parseArgs = (args) => {
           showError("Expected example env path but none found");
         }
         if (!existsSync(exampleEnvPath)) {
-          showError("Example env path does not exist: ", exampleEnvPath);
+          showError("Example env path does not exist: " + exampleEnvPath);
```

Comment:
There was a small bug. `exampleEnvPath` was not actually used since `showError` expects only one variable.